### DISCUSSION TITLE
Deploy to production server

### DISCUSSION THREAD

**Discussion by HomaEEE** [filamentphp/filament#3034] [Help] [answered]
**Package**
filament (admin panel, tables, forms)

**Package Version**
v2.13

**Laravel Version**
v9.19

**Livewire Version**
v2.10

**PHP Version**
PHP 8.1.0

**Node**
16

**Bug description**

Hello, I trying deploy to production server from local and GitHub and take some error
Lost assets and don't work route for Filament

**Steps to reproduce**
Copy local to vps and try pull from git
after import - `composer update && filament:upgrade`

**Comment by danharrin** [comment, marked as answer]
https://github.com/livewire/livewire/issues/242#issuecomment-948054946
